tests/rolling_deploy_test.py

```diff
@@ -215,6 +215,7 @@ def test_get_instance_ip_addrs(self):
   @mock_autoscaling
   @mock_elb
   def test_get_all_instance_ids(self):
+    self.setUpELB(env='stg')
```

Comment:
We already have env set to stg as a default in the method. You should be able to just run setUpELB() without passing in the stg environment.

```
  def setUpELB(self, env='stg'):
```